src/components/inputs/markdown/markdown.component.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import MarkdownWrapper from './markdown-wrapper.component';
 import { type FormFieldInputProps } from '../../../types';
 
 const Markdown: React.FC<FormFieldInputProps> = ({ field }) => {
-  return !field.isHidden && <MarkdownWrapper markdown={field.value} />;
+  const { t } = useTranslation();
+  return !field.isHidden && <MarkdownWrapper markdown={t(field.value)} />;
```

Comment:
I'm mostly concerned about issues that may arise with handling special characters, maybe we want to do something like:

```tsx
<MarkdownWrapper
        markdown={t(field.value, { defaultValue: field.value, interpolation: { escapeValue: false } })}
      />
```

cc: @ibacher

Reply:
@samuelmale Why would we want to disable special characters in this case? would they affect the way the strings are being translated?